test/box-luatest/gh_7255_box_info_before_cfg_test.lua

```diff
@@ -0,0 +1,8 @@
+local t = require('luatest')
+
+local g = t.group()
+
+g.test_box_info_before_box_cfg = function()
+    local info = box.info()
+    t.assert_equals(info.status, "not_configured")
+end
```

Comment:
BTW, it'll fail if we'll run it using luatest directly (without test-run) due to #7715. It shouldn't affect CI, but may be sometimes confusing. Not a blocker.

Reply:
Seems fixed now.